src/components/DropHover.tsx

```diff
@@ -70,19 +71,32 @@ const DropHoverIfVisible = ({
 }) => {
   // true if a thought is being dragged over this drop hover
   const showDropHover = useSelector(state => {
+    // Don't show drop hover on thoughts that are being dragged
+    const isThisThoughtBeingDragged = state.draggingThoughts.some(draggingPath => equalPath(draggingPath, simplePath))
+
+    if (isThisThoughtBeingDragged) return false
+
+    // Don't show drop hover between contiguous selected thoughts
+    const isDropBetweenSelected =
+      prevChildId &&
+      state.draggingThoughts.some(draggingPath =>
+        equalPath(draggingPath, appendToPath(parentOf(simplePath), prevChildId)),
+      )
+
+    if (isDropBetweenSelected) return false
+
     // Typically we show the drop hover if the thought is being directly hovered over.
     // However, when moving into a different context that is sorted, we need to show the drop hover on the sorted drop destination if the thought is hovered over any of the thoughts in the sorted context.
     const parentId = getThoughtById(state, head(simplePath))?.parentId
-    const sameContext =
-      state.draggingThought &&
-      equalPath(rootedParentOf(state, state.draggingThought), rootedParentOf(state, simplePath))
+    const sameContext = state.draggingThoughts.every(draggingPath =>
+      equalPath(rootedParentOf(state, draggingPath), rootedParentOf(state, simplePath)),
+    )
     const isParentSorted = parentId && getSortPreference(state, parentId).type === 'Alphabetical'
     if (!isParentSorted || sameContext) return testFlags.simulateDrag || isHovering
     else if (!state.dragInProgress) return false
 
-    const draggingThoughtValue = state.draggingThought
-      ? getThoughtById(state, head(state.draggingThought))?.value
-      : null
+    const draggingThoughtValues =
+      state.draggingThoughts.map(draggingPath => getThoughtById(state, head(draggingPath))?.value).filter(Boolean) || []
```

Comment:
`.filter(Boolean)` shouldn't be used on `string[]`, otherwise it will incorrectly filter out empty strings. 

Use `.filter(nonNull)` instead.